example-capabilities/src/main/java/mil/emp3/examples/capabilities/navItems/FreehandDraw.java

```diff
@@ -96,78 +83,56 @@ public boolean actOn(String userAction) {
                 }
                 testThread.interrupt();
             } else if(userAction.equals("ClearMap")) {
-                clearMaps();
-            } else if(userAction.equals("Remove Listeners")) {
-                freehandEventListener[whichMap] = null;
-                if(null != mapFreehandEventListenerHandle[whichMap]) {
-                    maps[whichMap].removeEventListener(mapFreehandEventListenerHandle[whichMap]);
-                    mapFreehandEventListener[whichMap] = null;
-                    mapFreehandEventListenerHandle[whichMap] = null;
+                for(int ii = 0; ii < ExecuteTest.MAX_MAPS; ii++) {
+                    if(null != maps[ii]) {
+                        try {
+                            maps[ii].drawFreehandExit();
+                        } catch(EMP_Exception e) {
+
```

Comment:
Logging is specifically not done. This is just a clean up because user either exited or executed clear map. Map may or may not be in the DRAW_MODE. So purposely not logging.